src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -949,3 +951,86 @@ def test_api_video_initiate_upload_staff_or_user(self):
             self.assertEqual(
                 content, {"detail": "Authentication credentials were not provided."}
             )
+
+    def test_api_video_get_detail_staff_or_user(self):
+        """Staff users should be able to get video details with a drf token. Regular user can't"""
```

Comment:
Split this test in tow separated. One testing a simple user and one testing a staff user.